docs/DeveloperGuide.md

```diff
@@ -446,171 +446,351 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `MyBookshelf` and the **Actor** is the `Community Library Manager (CLM)`, unless specified otherwise)
 
-#### Use case: UC1 - Add user to Contact List
-
+#### Use case: UC1 - Add user to the Contact List
 #### MSS:
-
 1.  User provides the following details:
     * Name
     * Phone number
     * Email
     * Address
     * Optionally, additional tags
-2. CLM enters the provided information.
+2. Library manager enters the provided information.
 3. MyBookshelf adds the user to the Contact List.
-4. MyBookshelf notifies CLM that the user has been successfully added.
+4. MyBookshelf notifies library manager that the user has been successfully added.
 
 ***Use case ends***
 
 #### Extensions:
-
 * 2a. MyBookshelf detects an error in the entered information.
-
     * 2a1. MyBookshelf requests for the valid information.
+    * 2a2. Library manager requests information from user.
+    * 2a3. Library manager enters new information.
+      Steps 2a1-2a3 are repeated until the information entered are valid.
 
-    * 2a2. CLM requests information from user.
+  ***Use case resumes from step 3***
 
-    * 2a3. CLM enters new information.
-    
-    Steps 2a1-2a3 are repeated until the information entered are valid.
+* *a. At any time, library manager chooses to cancel the addition of user.<br>
+    * *a1. Library manager clears the command line using the backspace key.<br>
 
-    ***Use case resumes from step 3.***
+  ***Use case ends***
 
-* *a. At any time, CLM chooses to cancel the addition of user.<br>
-
-  * *a1. CLM clears the command line using the backspace key.<br>
-  
-  ***Use case ends.***
 
 #### Use case: UC2 - List all users
-
 #### MSS:
-
-1. CLM intends to list all users.
-2. CLM enters the command.
+1. Library manager requests to list all users.
+2. Library manager enters the command.
 3. MyBookshelf retrieves the information from the Contact List.
 4. MyBookshelf displays a list of all users, including their names, contact information, and any other relevant details.
-5. CLM reviews the list of users.
+5. Library manager reviews the list of users.
 
 ***Use case ends***
 
 #### Extensions:
-* *a. At any time, CLM chooses to cancel the addition of user.<br>
-
-    * *a1. CLM clears the command line using the backspace key.<br>
+* *a. At any time, library manager chooses to cancel to list all users.<br>
+    * *a1. Library manager clears the command line using the backspace key.<br>
 
   ***Use case ends.***
 
+
 #### Use case: UC3 - Edit user's information
+#### MSS:
+1. Library manager requests to edit a user's information.
```

Comment:
I feel "Library manager **requests** to edit a user's information." should change to "Library manager **intends** to edit a user's information."